tests/python/relay/test_pipeline_executor.py

```diff
@@ -0,0 +1,251 @@
+
+    """The first model has three output."""
+
+    net1_output1 = relay.add(data, mv1)
+    net1_output2 = relay.subtract(data, mv2)
+    net1_output3 = relay.multiply(data, mv3)
+
+    """The second model use first model net1_output1 as first input,
+    here data_net1_output_1 represent net_output1, the second input
+    of this model is data21.
+    """
+    net2 = relay.add(data_net1_output_1, mv2)
+    net2 = relay.add(net2, data21)
+    net2_output = relay.add(net2, mv3)
+
+    """The third model use the second model net2_output as first input
+    and use the first model net1_output2 as second input.
+    """
+    net3 = relay.multiply(data_net2_output_1, mv3)
+    net3 = relay.add(net3, data_net1_output_2)
+
+    mods.append(
+        tvm.IRModule.from_expr(
+            relay.Function([data], relay.Tuple([net1_output1, net1_output2, net1_output3]))
+        )
+    )
+    mods.append(tvm.IRModule.from_expr(relay.Function([data_net1_output_1, data21], net2_output)))
+    mods.append(
+        tvm.IRModule.from_expr(relay.Function([data_net1_output_2, data_net2_output_1], net3))
+    )
+
+    return mods, dshape
+
+
+def get_manual_conf(mods, target):
+    """This function is used to generate manual pipeline configuration."""
+    mod_config = {}
+    """The third output is the final output, the second output is for mod3, the first is for mod2
+    input.
+    """
+    pipe_config1 = {
+        "mod_idx": 1,
+        "output": [
+            {"output_idx": 0, "dependent": [{"mod_idx": 2, "input_name": "data_0"}]},
+            {"output_idx": 1, "dependent": [{"mod_idx": 3, "input_name": "data_0"}]},
+            {"output_idx": 2, "dependent": [{"mod_idx": 0, "input_name": "0"}]},
+        ],
+    }
+    mod_config[mods[0]] = {
+        "pipeline": pipe_config1,
+        "target_host": None,
+        "mod_name": "default",
+        "build": None,
+        "params": None,
+        "target": target[0],
+        "dev": target[1],
+    }
+
+    pipe_config2 = {
+        "mod_idx": 2,
+        "output": [
+            {"output_idx": 0, "dependent": [{"mod_idx": 3, "input_name": "data_1"}]},
+        ],
+    }
+    mod_config[mods[1]] = {
+        "pipeline": pipe_config2,
+        "target_host": None,
+        "mod_name": "default",
+        "build": None,
+        "params": None,
+        "target": "llvm",
+        "dev": tvm.cpu(0),
+    }
+
+    pipe_config3 = {
+        "mod_idx": 3,
+        "output": [{"output_idx": 0, "dependent": [{"mod_idx": 0, "input_name": "1"}]}],
+    }
+    mod_config[mods[2]] = {
+        "pipeline": pipe_config3,
+        "target_host": None,
+        "mod_name": "default",
+        "build": None,
+        "params": None,
+        "target": "llvm",
+        "dev": tvm.cpu(0),
+    }
+    return mod_config
+
+
+def test_pipe_config_check():
+    """This function is used to trigger runtime error by appling wrong logic connection."""
+
+    """Get three pipeline modules here.
```

Comment:
For one line comment, just use #, not """. There are many of them in this function.

Reply:
fixed.